### DISCUSSION TITLE
Install brew on opt and usr

### DISCUSSION THREAD

**Discussion by bzh32** [Homebrew/discussions#3486] [Tap maintenance and Homebrew development]
I was trying to install an x86 package(pypy) on my m1 Mac. It greeted me with an unable to install on x86 error. Than, I installed rosetta and opened the terminal in rosetta, meaning it is emulating terminal in i386(intel). After trying again, it gave me this:
`Error: Cannot install under Rosetta 2 in ARM default prefix (/opt/homebrew)!
To rerun under ARM use:
    arch -arm64 brew install ...
To install under x86_64, install Homebrew into /usr/local.`
I tried to install home-brew with the command on brew.sh(website) and it attempted to install, but after resolving deltas, the brew froze for around 30min. I tried 5 times but it still did not work.

**Comment by SMillerDev** [comment]
> Than, I installed rosetta and opened the terminal in rosetta, meaning it is emulating terminal in i386(intel).

It's actually emulating `x86_64`.

So, what are the actual commands you ran with their output? Because "After trying again", "with the command on brew.sh(website)" and "I tried 5 times but it still did not work." are really hard to point to a specific problem
